pkg/datastore/ingest.go

```diff
@@ -144,6 +144,7 @@ func (s *Datastore) Ingest(ctx context.Context, datasetID string, name string, c
 	}
 
 	if ingestionFlow.Load == nil {
+		statusLog.With("status", "unsupported").Info(fmt.Sprintf("Unsupported file types: %s", filetype))
```

Comment:
To not introduce more log variations, I think this should be status=skipped with reason=unsupported (and I think we should have that log line somewhere already 🤔)

Reply:
Sounds good. 👍